system_metrics_collector/src/topic_statistics_collector/constants.hpp

```diff
@@ -25,6 +25,11 @@ namespace topic_statistics_constants
 constexpr const char kMsgAgeStatName[] = "message_age";
 constexpr const char kMsgPeriodStatName[] = "message_period";
 constexpr const char kMillisecondUnitName[] = "ms";
+
+constexpr const char kCollectStatsTopicNameParam[] = "collect_topic_name";
+constexpr const char kDefaultCollectStatsTopicName[] = "/topic";
```

Comment:
I don't really get this as a default, shouldn't it be a required parameter? It also occurs to me that someone may want to subscribe to multiple topics, so maybe we want a future feature to handle a list and have the node handle multiple subscriptions.

Reply:
It is a required parameter since it throws if empty. I removed this default value because it's not needed and created https://github.com/ros-tooling/system_metrics_collector/issues/118 to implement multiple subscriptions.